harper-core/src/linting/possessive_your.rs

```diff
@@ -65,6 +65,7 @@ mod tests {
     use super::PossessiveYour;
 
     #[test]
+    #[should_panic] // currently fails because comments is a homographs (verb or noun)
```

Comment:
Ooh handy! Didn't know about this. I'll remember to use it in other tests I wanted to include for things that don't work but should.